modules/undertoneBidAdapter.js

```diff
@@ -51,15 +61,24 @@ export const spec = {
   },
   buildRequests: function(validBidRequests, bidderRequest) {
     const payload = {
-      'x-ut-hb-params': []
+      'x-ut-hb-params': [],
+      'commons': {
+        'adapterVersion': $$PREBID_GLOBAL$$.version,
```

Comment:
Referencing $$PREBID_GLOBAL$$ is not permitted.  For version, please use '$prebid.version$' instead

Reply:
Thanks you, changed